tests/config/test_config.py

```diff
@@ -0,0 +1,95 @@
+"""Test config."""
+
+import os
+import tempfile
+
+import pytest
+from pydantic import ValidationError
+
+from ocf_data_sampler.config.load import load_yaml_configuration
+from ocf_data_sampler.config.model import Configuration
+from ocf_data_sampler.config.save import save_yaml_configuration
+
+
+def test_default():
+    """
+    Test default pydantic class
+    """
+
+    _ = Configuration()
+
+
+def test_yaml_load_test_config(test_config_filename):
+    """Test that yaml loading works for 'test_config.yaml'"""
+
+    config = load_yaml_configuration(test_config_filename)
+
+    assert isinstance(config, Configuration)
+
+
+def test_yaml_save():
+    """
+    Check a configuration can be saved to a yaml file
+    """
+
+    with tempfile.NamedTemporaryFile(suffix=".yaml") as fp:
+        filename = fp.name
+
+        # check that temp file can't be loaded
+        with pytest.raises(TypeError):
+            _ = load_yaml_configuration(filename)
```

Comment:
I'm not sure why we'd want this check. I think we should get rid of it

Reply:
yea, could strip it out, it does just check that we get an error properly when trying to load a file that doesnt have a config in it, but perhaps we can remove it for now, to keep it simple